src/applications/rated-disabilities/tests/components/RatedDisabilityList.unit.spec.jsx

```diff
@@ -9,7 +9,7 @@ describe('<RatedDisabilityList/>', () => {
   const ratedDisabilities = {
     ratedDisabilities: [
       {
-        decisionCode: 'SVCCONNECTED',
+        decisionCode: 'SVCCONNCTED',
```

Comment:
Weird that I see this spelled both ways in the code base...

Reply:
I looked at what EVSS actually sends back in staging and the vets-api code, but probably needs to be updated elsewhere as well

Reply:
I updated the other files that use the incorrect value